packages/ember-data/lib/system/store.js

```diff
@@ -440,9 +440,36 @@ Store = Ember.Object.extend({
     store.find('person', { page: 1 });
     ```
 
-    This will ask the adapter's `findQuery` method to find the records for
-    the query, and return a promise that will be resolved once the server
-    responds.
+    By passing an object {page: 1} as an argument to the find method, it
+    delegates to the adapter's findQuery method. The adapter then makes
+    a call to the server, transforming the object {page: 1} as parameters
```

Comment:
Lets surround the `{page: 1}` with backticks "`" to make this easier to read when rendered. 
![screen shot 2015-01-24 at 3 13 22 pm](https://cloud.githubusercontent.com/assets/54056/5888835/ac9bbd76-a3db-11e4-9fcb-3d5851353c05.png)